content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -144,6 +143,115 @@ version sort order is `v1`, followed by `v1beta1`. This causes the kubectl
 command to use `v1` as the default version unless the provided object specifies
 the version.
 
+## Webhook conversion
+
+The above example has a None converter between versions which only set `apiVersion` field
+on conversion and do not change the rest of the object. API server supports webhook conversion that
+calls an external service in case of a conversion is required, e.g. when:
+
+* CR is requested in a different version than stored version.
+* CR list request contains objects of different versions than the request version.
+* Watch is created in one version but the changed object is stored in another version.
+* CR PUT request is in a different version than storage version.
+
+To cover all of these cases and to optimize conversion on API Server side, the conversion requests
+comes with a list of objects to be converted. These conversions should happened independent on these
```

Comment:
"requests come" sound strange. Better: ..., the conversion requests carry as many object as possible in order to minimize the calls to the webhook.

Reply:
These conversions [or conversion request] should happen independently?

nit: The rest of the sentence could use an overhaul.